code/game/objects/effects/holy.dm

```diff
@@ -0,0 +1,36 @@
+//Effects used for anything holy and divine currently used for smite and spawming a admin dummy... maybe for holy explosion one day
```

Comment:
Oh so all content that is ported from pre rebase or not a tg port goes there? what about smite.dm? tg did not have smite.dm so I had to add that file back.